docs/reference-guides/best-practices/rancher-server/tips-for-scaling-rancher.md

```diff
@@ -0,0 +1,61 @@
+
+## General Tips on Optimizing Rancher's Performance
+* It is advisable to keep Rancher up to date with patch releases. Performance improvements and bug fixes are made throughout the life of a minor release. You can review the release notes to help inform your own decisions on whether an upgrade is necessary but we recommend keeping yourself up to date in most cases.
+
+* Performance will be negatively impacted by increased latency between Rancher's infrastructure and a downstream cluster's infrastructure (eg. geographic distance). If a user or organization requires clusters/nodes all over the world or spread across many regions, it is best to use multiple Rancher installations.
+
+* Please always try to scale up gradually, monitoring and observing any change in behavior while doing do. It is usually easier to resolve performance problems as soon as they surface, and before other problems confuse symptoms. 
+
+## Minimizing Load on the local cluster
+The largest bottleneck when scaling Rancher is resource growth in the local Kubernetes cluster. The local cluster contains information for all downstream clusters. Many operations that apply to downstream clusters will create new objects in the local cluster and require computation from handlers running in the local cluster.
+
+### Managing Your Object Counts
+ETCD eventually encounters limitations to the number of a single Kubernetes resource type it can store. These exact numbers are not well documented. From internal observations we usually see performance issues once a single resource type's object count exceeds 60k, and often that type is Rolebindings.
+
+Rolebindings are created in the local cluster as a side effect of many operations.
+
+Considerations when attempting reduce rolebindings in the local cluster:
+* Only add users to clusters and projects when necessary
+* Remove clusters and projects when they are no longer needed
+* Only use custom roles if necessary
+* Use as few rules as possible in custom roles
+* Consider whether adding a role to a user is redundant
+* Consider that using less, but more powerful, clusters may be more efficient
+* Monitoring the rolebindings created from creating new projects for specific purposes versus creating new clusters
+
+### Using Apps V2 Over Apps V1
+There are two App kubernetes resources that Rancher uses: apps.projects.cattle.io and correct this apps.cattle.cattle.io. V1 here refers to apps.projects.cattle.io since they were introduced first and are legacy, while V2 refers to apps.catalog.cattle.io. Apps V2 are preferrable because they live in the downstream cluster while Apps V1 lives in the local cluster. The dashboard uses Apps V2 while the old `/g/clusters` UI uses Apps V1.
+
+We recommend removing apps that appear in `/g/clusters`, replacing them with apps in the dashboard if necessary and creating any future apps in the dashboard only.
```

Comment:
```suggestion
We recommend removing apps that appear in Cluster Manager, replacing them with apps in the Cluster Explorer dashboard if necessary. Any new apps should be created in the dashboard only.
```

Reply:
I also clarified that the Cluster Explorer should be for the target cluster. Since the legacy apps are installed in local technically.